test/index.test.mjs

```diff
@@ -62,3 +63,77 @@ describe('percySnapshot', () => {
     ]));
   });
 });
+
+describe('percyScreenshot', () => {
```

Comment:
Tests are failing and coverage is not 100%

Reply:
![Screenshot 2023-05-29 at 6 07 25 PM](https://github.com/percy/percy-selenium-js/assets/92926953/b192354e-0bdf-4099-9ff1-21483839303b)
It is due to CLI changes not merged. getting 100% in local